modules/openy_features/openy_prgf/modules/openy_prgf_4c/openy_prgf_4c.info.yml

```diff
@@ -3,6 +3,7 @@ description: 'Implements a paragraph with 4 columns.'
 type: module
 core: 8.x
 package: OpenY
+version: '8.x-1.0'
 dependencies:
```

Comment:
This module has a module `block_content ` in the list of dependencies, but seems it should not has it. Please check.

Reply:
I have replaced this dependency by `openy_block_custom_simple` (and `openy_block_custom_simple` depend on `block_content`)